core/datapool/datapool-core/src/test/kotlin/io/holunda/polyflow/datapool/core/business/DataEntryAggregateTest.kt

```diff
@@ -164,6 +164,57 @@ class DataEntryAggregateTest {
       )
   }
 
+  @Test
+  fun `should anonymize aggregate`() {
```

Comment:
This test is good. I'm missing a second one responsible for rejecting anonymisation of the deleted aggregate. Please add it.

Reply:
done